.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
```

Comment:
I suppose this file has to be in the root directory? The root clutter is annoying -- then again, I don't see a much better place to put it.

Reply:
As far as I know, yes. Since it's language-agnostic, it doesn't support going into something like pyproject.toml, and indeed there doesn't really seem like anywhere else it could go.